docs/docs/getting-started/first-spec.md

```diff
@@ -101,7 +101,7 @@ val create: int -> t
 
 Now on to `is_empty` and `mem`.
 
-`is_empty t` is true if and only if `t` is empty; this is a post-condition. This
+`is_empty t` returns true if and only if `t` is empty; this is a post-condition. This
```

Comment:
`is_empty t` does not return `true`, it _is_ `true` :) Can you revert just that bit?

Reply:
Hm. I agree that it is standard terminology to say a *function* returns some value. However it is not unusual to say that a *function call* returns some value, see, e.g., 
https://docs.microsoft.com/en-us/dotnet/fsharp/introduction-to-functional-programming/first-class-functions#return-the-value-from-a-function-call  for someone else in FP also using this terminology.
I guess it depends on whether you consider the call as completed or not. Operationally, when the call hasn't been carried out or hasn't completed yet there is no return value to talk of. For an application in logic this is of course a different matter...
Anyway, I reverted the change as you already saw.